src/yb/server/server_base.cc

```diff
@@ -458,6 +458,7 @@ void RpcAndWebServerBase::GenerateInstanceID() {
   // TODO: maybe actually bump a sequence number on local disk instead of
   // using time.
   instance_pb_->set_instance_seqno(Env::Default()->NowMicros());
+  instance_pb_->set_start_time(Env::Default()->NowMicros());
```

Comment:
Should the value of Env::Default()->NowMicros() in the previous call be saved and used here ?

Reply:
@tedyu  ,  As  @bmatican  said :
  `Note, that even though technically, I think we could already use the instance_seqno, because it is implemented as the NowMicros of when the server starts up, that feels like an implementation detail that we should not rely on...`

Reply:
@tedyu  Thanks for your review !

Reply:
@tedyu  Thanks for your review !

Reply:
Ack.

Reply:
I think Ted's comment is still fair, just create a new `auto now = ...` and use it in both the spots, that way we don't compute time twice. The comment still remains valid, and if we ever address the TODO, we'd change this.